### PR TITLE
Replaced custom docker image for PostgreSQL with the official one

### PR DESCRIPTION
**Pull request by morozov**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | [job #488051004](https://travis-ci.org/doctrine/dbal/jobs/488051004)

### CODE REVIEW

**Review by lcobucci — APPROVED** [review]
LGTM, it's much nicer to not have to build the image during the build!

File: tests/travis/install-postgres-11.sh [hunk 1]
```diff
@@ -6,9 +6,7 @@ echo "Preparing Postgres 11"
 
 sudo service postgresql stop || true
 
-sudo docker build -t postgres11 - < tests/travis/Dockerfile-postgres11
-sudo docker run -d --name postgres11 -p 5432:5432 postgres11
-sudo docker exec postgres11 service postgresql start
-sudo docker exec -i postgres11 su -c psql postgres <<<"create database doctrine_tests"
+sudo docker run -d --name postgres11 -p 5432:5432 postgres:11.1
```
[the remark this answers is not in the document]
Comment (morozov) [reply tests/travis/install-postgres-11.sh]: We can try using it if it shows visible performance improvement. Right now, I just chose the most default one to fix the build failure.

[the remark this answers is not in the document]
Comment (Majkl578) [reply tests/travis/install-postgres-11.sh]: I always recommend against using Alpine-based images, they use different libc which causes problems for locale-based stuff.

[the remark this answers is not in the document]
Comment (lcobucci) [reply tests/travis/install-postgres-11.sh]: @Majkl578 interesting, never had any issue 🤔

[the remark this answers is not in the document]
Comment (lcobucci) [reply tests/travis/install-postgres-11.sh]: @Majkl578 fun!

**Pull request merged**